content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -15,39 +15,29 @@ Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites)
 
 ## Installation steps
 
-1. If using a Helm version prior to 2.10.0, install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed to the Kubernetes API server:
+Choose one of the following two **mutually exclusive** options described below.
 
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
-    {{< /text >}}
+### Option 1: Install with Helm via `helm template`
 
-1. Choose one of the following two
-**mutually exclusive** options described below.
+Choose this option if your cluster doesn't have [Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components) deployed and you don't want to install it.
 
-### Option 1: Install with Helm via `helm template`
+1. Install all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kube api-server:
+
+    {{< text bash >}}
+    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
```

Comment:
Is this approach really the right way to do it without helm? These CRDs have labels `heritage: Tiller` which makes no sense if Tiller is not used.

Shouldn't we have a build step that generates a non-helm yaml file (e.g., `istio-crds.yaml` that non-helm installs use. Don't we need that for users that install `istio-demo.yaml` anyway? https://preliminary.istio.io/docs/setup/kubernetes/quick-start/#option-1-install-istio-with-mutual-tls-enabled-and-set-to-use-permissive-mode-between-sidecars

Reply:
@sdake Any thought for this?

Reply:
@frankbu Maybe we can open another PR for this, what do you think?

Reply:
@frankbu currently istio-init.yaml is created by the build system.  In addition, when istio-demo.yaml is generated, it is a cat of namespace.yaml, the CRDs, followed by istio-demo.yaml.

We should do an audit of labels that were unnecessarily added to all of the resources in 1.1 and remove them.  @costinm had suggested this some time ago, but the work hasn't been done.  Once the labels are in, they cannot be removed as the label field in a spec is immutable and my understanding is labels are expensive in Kubernetes.

Reply:
@sdake, I guess you're saying that these are the correct instructions to install CRDs for the helm w/o tiller option. The istio-demo.yaml case is handled by istio-demo.yaml itself.

Is there an issue for removing `heritage: Tiller` (and any other unnecessary labels) from the templates? If I understand @sdake, it sounds like a must-have for 1.1.

Reply:
@frankbu Right on.  As far as labels, we can't remove all of them - only the new ones that were introduced in release-1.1 branch.  Any present in the 1.0 branch need to stay, as labels are immutable and we wouldn't be able to upgrade.

I am unclear on the CRD labels - but I think the same applies.

Cheers
-steve

Reply:
Is somebody signed up to send a PR to remove these labels? @morvencao?

Reply:
@frankbu @sdake Since the labels can't be removed for hem /w tiller method, while they can for `kubectl apply ...` method. Can we add condition that only remove the labels for `kubectl apply...` and keep them for helm /w tiller?

Reply:
@sdake @frankbu Have raise a PR: https://github.com/istio/istio/pull/11301 for you review.